### DISCUSSION TITLE
Secondo voi quanto tempo ci avremmo messo se nico non fosse stato in gruppo con noi?

### DISCUSSION THREAD

**Discussion by TomPaglia** [Frati-Paoloni/Azienda_Vitivinicola#101] [Q&A] [answered]
ESPRIMETE IL VOSTRO PARERE

**Comment by RizziLucaH** [comment, marked as answer]
Ciao Tommaso, stimo una riduzione del 43% del tempo

**Comment by dade04** [comment]
Ciao, secondo i miei calcoli il progetto poteva essere terminato il 10 aprile.

**Comment by Nicolas-FerrariH** [comment]
ripeto la mia idea a riguardo (dato che sono stato censurato più volte) tutto ciò è un vero e proprio caso di bullismo non lo apprezzo
